src/index.js

```diff
@@ -1,6 +1,18 @@
 const fibonacci = (n) => {
-
+  let fibArray = [];
+  if(n === 1){
+    fibArray.push(1)
+  }
+  for(let i = 1; i < n; i++){
+      if(i <= 1){
+        fibArray.push(1);
+      }
+      const FirstlastNum = fibArray[i - 1]
+      const SecondLastNum = fibArray[i - 2] === undefined ? 0 : fibArray[i - 2];
```

Comment:
Use camel case , for example `firstLastNumber`. Use first letter in capital letter when you write a class, for example `class MyClass {}`